x-pack/plugins/cloud_security_posture/server/plugin.ts

```diff
@@ -209,6 +213,7 @@ export class CspPlugin
     await initializeCspIndices(esClient, this.config, this.logger);
     await initializeCspTransforms(esClient, this.logger);
     await scheduleFindingsStatsTask(taskManager, this.logger);
+    await getCspSettingObjectSafe(core.savedObjects.createInternalRepository(), this.logger);
```

Comment:
It is not clear why getting the settings is relevant to the plugin's initialization